cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/aws-orchestrator/AwsOrchestrator.service.tsx

```diff
@@ -41,3 +64,24 @@ function setDefaultOsImage(props: AwsOrchestratorProps, osImages: AWSVirtualMach
         payload: AwsDefaults.selectDefalutOsImage(osImages),
     } as FormAction);
 }
+
+function setDefaultEC2KeyPair(props: AwsOrchestratorProps, keyPairs: AWSKeyPair[]) {
+    props.awsDispatch({
+        type: INPUT_CHANGE,
+        field: AWS_FIELDS.EC2_KEY_PAIR,
+        payload: AwsDefaults.selectDefalutEC2KeyPairs(keyPairs),
+    } as FormAction);
+}
+
+function removeErrorInfo(errorMessage: { [key: string]: any }, field: AWS_FIELDS) {
```

Comment:
Love these error message methods!
My same nit as above: I'd suggest `errorObject` as the param name instead of `errorMessage`